src/field/vector3d.cxx

```diff
@@ -603,6 +603,34 @@ const Field3D abs(const Vector3D &v, const std::string& region) {
   return sqrt(v*v, region);
 }
 
+const Vector3D toFieldAligned(const Vector3D& v, const std::string& region) {
+  Vector3D result;
+  result.setLocation(v.getLocation());
+  if (not v.covariant) {
+    result.toContravariant();
```

Comment:
We probably should have an `emptyFrom` and `zeroFrom` for vectors, like for fields.